### PR TITLE
Added info on jvm backends to compatibility guide

### PR DESCRIPTION
**Pull request by danil-pavlov**
update: compatibility guide updated with a note about switching to the old jvm backend

### CODE REVIEW

File: docs/topics/compatibility-guide-15.md [hunk 1]
```diff
@@ -150,6 +150,55 @@ perspective
...
+> - \>= 1.5: by default, the IR-based backend is used. If you need to use the old backend in Kotlin 1.5, add
+> the following lines to the project’s configuration file to temporarily revert to pre-1.5 behavior:
+>
+> In Gradle:
+>
+> <tabs>
+>
+> ```kotlin
+> tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile> {
+>  kotlinOptions.useOldBackend = true
```
Comment (koshachy) [inline docs/topics/compatibility-guide-15.md:180]: indents?

File: docs/topics/compatibility-guide-15.md [hunk 2]
```diff
@@ -150,6 +150,55 @@ perspective
...
+> }
+> ```
+>
+> ```groovy
+> tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile) {
+> kotlinOptions.useOldBackend = true
```
Comment (koshachy) [inline docs/topics/compatibility-guide-15.md:186]: same

**Review by koshachy — APPROVED** [review]

File: docs/topics/compatibility-guide-15.md (hunk 1 above)
Comment (danil-pavlov) [reply docs/topics/compatibility-guide-15.md]: Thank you for noticing! fixed

File: docs/topics/compatibility-guide-15.md (hunk 2 above)
Comment (danil-pavlov) [reply docs/topics/compatibility-guide-15.md]: Thank you for noticing! fixed

**Pull request merged**